test/e2e/aws_iam_policy.jsonc

```diff
@@ -0,0 +1,219 @@
+/* AWS IAM policy for e2e tests.
+
+   Policy ARN:  arn:aws:iam::043455440429:policy/EndToEndTestSuite
+   Description: Set of permissions required by the TriggerMesh end-to-end test suite to run tests that manipulate AWS resources.
+*/
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "E2EFrameworkEventBridgeEventBuses",
+            "Effect": "Allow",
+            "Action": [
+                "events:CreateEventBus",
+                "events:DeleteEventBus",
+                "events:TagResource"
+            ],
+            "Resource": [
+                "arn:aws:events:*:043455440429:event-bus/aws.partner/triggermesh.com/*",
+                "arn:aws:events:*:043455440429:event-bus/aws.partner/triggermesh.com.test/*"
+            ]
+        },
+        {
+            "Sid": "E2EFrameworkEventBridgeRules",
```

Comment:
As mentioned before, some permissions are still not required but I'm in to keep them here soon. EventBridge is a bit special since that component it is not part of this repo.

I would keep it also in while we decide where EventBridge e2e tests should live.

Reply:
Very good point, didn't think about EventBridge.
I'll remove that one.

Reply:
Done in [78e34548..3ef0015a](https://github.com/triggermesh/triggermesh/compare/78e3454850a5701a7be2da7003b1cbef133c8434..3ef0015a2ac139077c6f183985dfb8066b734bd2)